EasyInventory/src/main/java/com/easyinventory/enterprise/service/EasyInventoryServiceIMPL.java

```diff
@@ -13,9 +15,11 @@
 @Service
 public class EasyInventoryServiceIMPL implements EasyInventoryService{
 
-    ItemRepository itemRepository;
+    private ItemRepository itemRepository;
 
-    CategoryRepository categoryRepository;
+    private CategoryRepository categoryRepository;
+
+    private Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
Implemented logging to handle/debug exceptions

Reply:
Good.